website/content/docs/sync/gcpsm.mdx

```diff
@@ -264,6 +264,52 @@ This overrides the project ID derived from the service account JSON credentials
 default credentials. The service account must be [authorized](https://cloud.google.com/iam/docs/service-account-overview#locations)
 to perform Secret Manager actions in the target project.
 
+## Access management
+
+You can allow or restrict access to secrets based on
+[IAM conditions](https://cloud.google.com/iam/docs/conditions-resource-attributes#resource-name)
+against the fully-qualified resource name. For secrets in Secret Manager, a fully-qualified resource name must have the following
+format:
+
+  `projects/<project_number>/secrets/<secret_name>`
+
+<Tip title="Use the project number, not the project ID"> 
+
+  The project **number** is not the same as the project **ID**. Project numbers
+  are **numeric** while project IDs are **alphanumeric**. They can be found on
+  the Project info panel in the web dashboard or on the Welcome screen.
+
+</Tip>
+
+For example, the default secret name template prepends the word `vault` to the
+beginning of secret names. To prevent Vault from modifying secrets that were not
+created by a sync operation, you can use a role binding against the resource
+name with the `startsWith` condition:
+
+  <CodeBlockConfig hideClipboard>
+
+    resource.name.startsWith("projects/<project_number>/secrets/vault")
+
+  </CodeBlockConfig>
+
+To prevent out-of-band overwrites, simply add a negative condition with `!` on any
+write-access role bindings not being used by Vault that contain Secret Manager permissions:
+
+  <CodeBlockConfig hideClipboard>
+
+    !(resource.name.startsWith("projects/<project_number>/secrets/vault"))
+
+  </CodeBlockConfig>
+
+To add these conditions and others to IAM principals in GCP, refer to the image below and the 
+[IAM Conditions overview documentation](https://cloud.google.com/iam/docs/conditions-overview).
+
+![GCP IAM Conditions](/img/gcp-add-iam-conditions_light.png#light-theme-only)
+![GCP IAM Conditions](/img/gcp-add-iam-conditions_dark.png#dark-theme-only)
+
+![GCP IAM Conditions](/img/gcp-add-iam-conditions_light2.png#light-theme-only)
+![GCP IAM Conditions](/img/gcp-add-iam-conditions_dark2.png#dark-theme-only)
```

Comment:
Do you like the addition of this statement along with a screenshot showing where IAM Conditions can be added? @schavis 

If so, the circled version or uncircled version?

Reply:
If we add an image, definitely use the circled one. But we need more useful alt text so it's meaningful for screen readers as well.